### DISCUSSION TITLE
Globalisation by POCO

### DISCUSSION THREAD

**Discussion by BrinthaShanmugam** [FirelyTeam/firely-net-sdk#1915] [Q&A] [answered]
Hi Team,
We are having a scenario where the value is in localized format(Ex: instead of 7.5 it is 7,5) . When we are using the toPOCO function (Ex: if finalNode is my source node to convert then the syntax which I am using is finalNode.ToPoco(ModelInfo.GetTypeForFhirType(resourceType))) to convert it is giving an error(Error details:System.FormatException: Type checking the data: Literal '7,5' cannot be interpreted as a decimal: 'Input string was not in a correct format.'. (at Observation.referenceRange[0].low[0].value[0]). Is there a way in toPOCO function itself which converts localized value into globalized value before transformation . If so can someone please help me in writing that function

**Comment by marcovisserFurore** [comment, marked as answer]
Hi @BrinthaShanmugam ,

The FHIR standard requires the datatype `decimal` to be written with a period (`.`). So you have to make sure that your input is already in the correct format. Firely .NET SDK does not have functionality to automatically convert that to the correct format.

**Reply by BrinthaShanmugam** [reply]
Thankyou so much for your input
